src/core/Pagination/index.stories.tsx

```diff
@@ -44,6 +44,14 @@ export default {
     },
   },
   component: Pagination,
+  parameters: {
+    axe: {
+      disabledRules: [
+        // FIXME - component has invalid nesting of non-LI elements within its UL.
+        "list",
```

Comment:
The Pagination has a structure like

```html
<ul>
  <button>
  <li>
  <li>
  <li>
  <button>
</ul>
```

which axe doesn't like.

Reply:
I'll fix this in a separate PR 🙇🏻‍♂️

Reply:
https://github.com/chanzuckerberg/sci-components/pull/298